src/main.cpp

```diff
@@ -6601,7 +6601,7 @@ void static ProcessGetData(CNode* pfrom)
                     if (inv.type == MSG_BLOCK)
                     {
                         LogPrint("forks", "%s():%d - Pushing block [%s]\n", __func__, __LINE__, block.GetHash().ToString() );
-                        pfrom->PushMessage("block", block);
+                        pfrom->PushMessage(NetMsgType::BLOCK, block);
                     }
                     else // MSG_FILTERED_BLOCK)
                     if (inv.type == MSG_FILTERED_BLOCK)
```

Comment:
Probably we want to modify also line 6613 (`pfrom->PushMessage("merkleblock", merkleBlock);`) leveraging `NetMsgType` (also need to add the `"merkleblock"` command in `protocol.h`/`protocol.cpp`).

Reply:
Yes, nice catch!